.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,13 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
We should add .idea folder to .gitignore all together.
We may add a editor config later to grant every IDE is configured as we want https://editorconfig.org/

Reply:
- [X] Done